src/main/java/uk/gov/hmcts/dts/fact/model/AdditionalLink.java

```diff
@@ -10,9 +12,17 @@
 public class AdditionalLink {
     private String url;
     private String description;
+    private String location;
 
     public AdditionalLink(uk.gov.hmcts.dts.fact.entity.AdditionalLink additionalLink) {
         this.url = additionalLink.getUrl();
         this.description = chooseString(additionalLink.getDescriptionCy(), additionalLink.getDescription());
+
+        final SidebarLocation location = additionalLink.getLocation();
+        if (location != null && StringUtils.isNotBlank(location.getName())) {
+            this.location = location.getName();
+        } else {
+            this.location = "";
```

Comment:
If location is going to be a different part on the frontend, should this be an enumerator? I.e location.getName refers (i imagine) to either one or the other. Might not be worth it, ignore if it isnt haha

Reply:
location.getName() is just the string stored in the database. I am not sure how to store an enum in the database as it is done through the flyway transformation. Do you mean returning the field as an enum from the endpoint instead? It currently returns the location field as a string, but the front-end will get the string and convert to its enum. Let me know if there is a better way to do this.